### PR TITLE
Add the fortune-teller game

### PR DESCRIPTION
**Pull request by Janhavi-2001**
**This PR is related to**:  A simple Python code implementing a fortune-telling game via the generation of a random number.

### CODE REVIEW

**Comment by pokacyb** [comment]
Hello!
It's my first pull request but I can try working on it if that's ok :)

File: Programs/fortune_teller.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import random
+
+print("Welcome to your fortune teller! Feel free to ask any question. Remember, no inappropriate questions!")
+while(True):
+    choice = str(input("Do you wish to continue? (Y\y or N/n): "))
+    if(choice == 'Y' or choice == 'y'):
```
Comment (Akshu-on-github) [inline Programs/fortune_teller.py]: Could you use this instead?
```py
if (choice.lower() == 'y'):
    # code block
```

File: Programs/fortune_teller.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        elif(n==6):
+            print("Never\n")
+        elif(n==7):
+            print("There are high chances!\n")
+        elif(n==8):
+            print("You may conclude so.\n")
+        else:
+            break
+
+    elif(choice == 'N' or choice == 'n'):
```
Comment (Akshu-on-github) [inline Programs/fortune_teller.py]: Here too?

**Review by Akshu-on-github — CHANGES_REQUESTED** [review]
Thanks for contributing! Lemme know if you're up for making these changes @Janhavi-2001

**Comment by Akshu-on-github** [comment]
@pokacyb - I think this PR is almost done, would you like to work on something else?

**Comment by Janhavi-2001** [comment]
I've made the changes as per your requirements. Feel free to check them out!

**Comment by Akshu-on-github** [comment]
LGTM, thanks for contributing!

**Review by Akshu-on-github — APPROVED** [review]
LGTM ✅ 
Thanks for making the requested changes!

**Comment by pokacyb** [comment]
@Akshu-on-github 
Github didn't send me a notification for your message for some reason.
Sure I'll do anything :)

**Pull request merged**